### PR TITLE
#5183 Add locale strings for legends in forms

### PR DESCRIPTION
**Pull request by mylonelycomputer**
From this issue: https://github.com/pkp/pkp-lib/issues/5183

### CODE REVIEW

File: locale/en_US/locale.po [hunk 1]
```diff
@@ -1240,6 +1246,9 @@ msgstr "Identify reviewing interests (substantive areas and research methods):"
 msgid "user.register.form.emailConsentAllContexts"
 msgstr "Yes, I would like to be notified of new publications and announcements for all journals that I registered with."
 
+msgid "user.register.form.privacyConsentLabel"
+msgstr "Option to consent"
```
Comment (NateWr) [inline locale/en_US/locale.po:1250]: Let's use "Privacy Options".

File: templates/frontend/pages/purchaseIndividualSubscription.tpl [hunk 2]
```diff
@@ -15,6 +15,9 @@
 		{csrf}
 
 		<fieldset>
+			<legend>
+				{translate key="user.subscriptions.form.typeId"}
```
Comment (NateWr) [inline templates/frontend/pages/purchaseIndividualSubscription.tpl:19]: Use `payment.subscription.purchase` ("Purchase Subscription")

File: templates/frontend/pages/purchaseInstitutionalSubscription.tpl [hunk 3]
```diff
@@ -25,6 +25,9 @@
 		{include file="common/formErrors.tpl"}
 
 		<fieldset>
+			<legend>
+				{translate key="user.subscriptions.form.typeId"}
```
Comment (NateWr) [inline templates/frontend/pages/purchaseInstitutionalSubscription.tpl:29]: Use `payment.subscription.purchase` ("Purchase Subscription")

**Review by NateWr — CHANGES_REQUESTED** [review]
Looks good. I left a few comments. In addition, let's move the new locale keys into pkp-lib. They can go into `common.po`. 

With that change, you shouldn't need any PRs for OMP or OPS. But go ahead and test the pkp-lib changes in OMP and OPS just to make sure.

**Comment by thinkbulecount2** [comment]
cc: @NateWr

File: locale/en_US/locale.po [hunk 4]
```diff
@@ -1240,6 +1246,9 @@ msgstr "Identify reviewing interests (substantive areas and research methods):"
...
+
```
Comment (NateWr) [inline locale/en_US/locale.po:1251]: All of the new locale keys in this PR are available in pkp-lib so they can be removed here.

File: templates/frontend/pages/purchaseIndividualSubscription.tpl [hunk 5]
```diff
@@ -15,11 +15,14 @@
...
 		<fieldset>
+			<legend>
+				{translate key="payment.subscription.purchase"}
+			</legend>
 			<div class="fields">
 				<div class="subscription_type">
 					<label>
 						<span class="label">
-							{translate key="user.subscriptions.form.typeId"}
+							{translate key="payment.subscription.purchase"}
```
Comment (NateWr) [inline templates/frontend/pages/purchaseIndividualSubscription.tpl:25]: I think this should stay as `user.subscriptions.form.typeId`. That would make the legend "Purchase Subscription" and this particular field "Subscription Type".

File: templates/frontend/pages/purchaseInstitutionalSubscription.tpl (hunk 3 above)
Comment (NateWr) [reply templates/frontend/pages/purchaseInstitutionalSubscription.tpl:29]: It looks like this still hasn't been done.

**Comment by thinkbulecount2** [comment]
Made changes, @NateWr.

**Pull request merged**